changelog.json

```
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "3.3.10-beta1": [
+      "[Added] Added visible and semantic check all button for a group of consecutive selectable rows in the diff - #18201",
+      "[Fixed] Ignore a files parent directories on Windows - #18206",
```

Should we fix this typo?

```suggestion
      "[Fixed] Ignore a file's parent directories on Windows - #18206",
```